optimum/onnxruntime/modeling_decoder.py

```diff
@@ -147,6 +147,7 @@ def __init__(
         self.generation_config = generation_config
         self.onnx_paths = [self.model_path]
         self.use_merged = "use_cache_branch" in self.inputs_names
+        self.model_type = self.config.model_type
```

Comment:
small remark but might make sense to keep a convention for dash / underscore in model types in the future :
```suggestion
        self.model_type = self.config.model_type.replace("_", "-")
```
https://github.com/huggingface/optimum/blob/554e3121b5cb1be6125ea41a2a89f46805c88c1f/optimum/onnxruntime/utils.py#L58

https://github.com/huggingface/optimum/blob/554e3121b5cb1be6125ea41a2a89f46805c88c1f/optimum/utils/modeling_utils.py#L21